README.md

```diff
@@ -1 +1,50 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+In this lesson we're going to be looking into AngularJS (commonly referred to as just "Angular"), an open-source framework that was released in 2009. It is currently being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
+
+## Objectives
+
+- Describe what Angular's role in the web platform is
+- Describe the issues Angular looks to solve
+
+## Frontend Frameworks
+
+Ever wondered how Facebook can have a different timeline for millions of users? How Twitter can display tweets for 300 million people? That's where frontend frameworks come in. 
```

Comment:
Really nice tone here - very friendly and eases the reader into the topic with real world examples. Could you include a specific example of how front-end frameworks improve user experience? ie "How can Gmail let you send and receive emails, star messages, view contacts, and chat with your friends all without ever reloading the page?" Use specific examples that the reader will be familiar with along with second-person pronouns.